doc/stake_credential_history_tool.md

```diff
@@ -1,30 +1,29 @@
 # Stake Credential History Tool
 
 
-Stake Credential History Tool is a very useful utility that produces a linear history of events for a given stake credential.
+The Stake Credential History Tool is a very useful utility that produces a linear history of events for a given stake credential.
```

Comment:
The stake credential history tool is...

Reply:
The page that the link leads to has the tool name capitalized in the heading but not the text.